### PR TITLE
Bug Fixed ZK-4051: scrollable tabbox tabs breaking in classic themes

### PR DESCRIPTION
**Pull request by wenninghsu**

### CODE REVIEW

File: zul/src/archive/web/js/zul/tab/Tabs.js [hunk 1]
```diff
@@ -193,6 +193,7 @@ zul.tab.Tabs = zk.$extends(zul.Widget, {
 					this._doScroll(d >= 0 ? 'right' : 'left', d >= 0 ? d : Math.abs(d));
 					break;
 				case 'init':
+				case 'sel':
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/tab/Tabs.js]: extra case

**Comment by charlesqqq** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**